### ISSUE TITLE
3D Convolution?

### ISSUE THREAD

**Issue by jingpengw** [pytorch/tvm#146]
It seems that there is no 3D convolution support. the tests are 1D and 2D. Is it correct? if so, are there plan to add 3D support? 3D convolution is popular in medical image processing now.

**Comment by yinghai** [comment]
Yeah, we currently don't have plans for that. If you are interested, please add a patch. :)
